### ISSUE TITLE
What is the input channel in the Set5 / Set14 test in the paper? RGB or Y?

### ISSUE THREAD

**Issue by forseeking** [LimBee/NTIRE2017#10]
What is the input channel in the Set5 / Set14 test in the paper? RGB or Y? Thanks for your answer.

**Comment by limbee** [comment]
Y channel for 4 conventional benchmark datasets (Set5, Set14, B100, Urban100), and RGB for DIV2K validation set. Sorry for the confusion.

**Comment by forseeking** [comment]
Thanks a lot. So the input to the neural network is only the Y channel for the (Set5, Set14, B100, Urban100) test.  Are they all trained with the DIV2K dataset?

**Comment by limbee** [comment]
I'm sorry, I just gave you the wrong answer. 

Because our model only works with 3-channel inputs, all inputs and outputs have RGB channels. 
However, when evaluating the PSNR and SSIM, only Y channel was used for 4 benchmarks, and RGB channels were used for DIV2K valset.

All models were trained on DIV2K training set (800 images) using RGB channels.

**Comment by forseeking** [comment]
Noted with thanks！

**Comment by luomomo** [comment]
I have use RGB channels  and Y channel to calculate the psnr, the psnr is 35.98 and 32 indenpendently. (DIV2K bicubic x2 validation 100 images).  Higher to 35.03 in the paper

**Comment by pzz2011** [comment]
Does 800 images cause overfitting?
It seems that 800 images are few。

**Comment by limbee** [comment]
We observed overfitting with EDSR, when we didn't use x8 geometric augmentation.

**Comment by qingchuanhuajuan** [comment]
Hi， LimBee,Your work is really great! There is a question I would like to ask you: the model trains the images of the three channels of RGB, so the output picture is also three channels. My problem is that there is a picture in the Set14 "bridge" is a single-channel picture, so the test results of the Set14 provided by you include this picture? If the number of test pictures is 14, then what should I do? Thank you!

**Comment by limbee** [comment]
@qingchuanhuajuan Yes we included that picture, by concatenating the image 3 times to make 3-channel image.
